tests/jax/test_custom_call_compute.py

```diff
@@ -294,6 +432,96 @@ def random_inputs_fixture(shape):
     return out
 
 
+class TestGeLu:
+
+    def ref_func(self, inputs):
+
+        func = jit(value_and_grad(lambda x: jnp.mean(jax.nn.gelu(x))))
+        return func(inputs)
+
+    def prim_func(self, inputs):
+
+        @jax.custom_vjp
+        def primitive(x):
+            out, _ = primitive_fwd(x)
+            return out
+
+        def primitive_fwd(x):
+            out = gelu(x)
+            ctx = x
+            return out, ctx
+
+        def primitive_bwd(ctx, g):
+            x = ctx
+            out = dgelu(g, x)
+            return (out,)
+
+        primitive.defvjp(primitive_fwd, primitive_bwd)
```

Comment:
Why create this in the test file? Shouldn't this be in another file? Some ends user will want to use that gelu functional API.

Reply:
This VJP function is for testing purpose only. It is kind of a simplier one of functions in `mlp.py`. We keep this to align the workflow of this test with others.